api/api_test.go

```diff
@@ -332,6 +334,80 @@ func TestUpsertTeam(t *testing.T) {
 	})
 }
 
+func TestGetTeamName(t *testing.T) {
+	t.Run("parses the name from a good response", func(t *testing.T) {
+		client, mux, _, teardown := setup()
+		defer teardown()
+
+		teamUUID := uuid.Must(uuid.NewV4())
+		teamResponse, err := json.Marshal(v1structs.GetTeamResponse{
+			Name: "Kiffix Ltd",
+		})
+		if err != nil {
```

Comment:
i find `assert.NoError` more helpful in cases like this one, since it's 1 line rather than 3, and it doesn't swallow the content of the error